### PR TITLE
LRQA-74419 | master

### PR DESCRIPTION
**Pull request by melodywu**
https://issues.liferay.com/browse/LRQA-74419

### CODE REVIEW

File: modules/apps/account/account-test/src/testFunctional/tests/AccountRoles.testcase [hunk 1]
```diff
@@ -188,4 +188,91 @@ definition {
...
+					resourceName = "com.liferay.account.model.AccountEntry",
+					roleTitle = "Account Specific Role",
+					roleType = "account");
+			}
+
+			Permissions.definePermissionViaJSONAPI(
+				resourceAction = "UPDATE",
+				resourceName = "com.liferay.account.model.AccountRole",
+				roleTitle = "Account Specific Role",
+				roleType = "account");
```
Comment (patriciajeanperez) [inline modules/apps/account/account-test/src/testFunctional/tests/AccountRoles.testcase:244]: @melodywu Same comment in previous PR, I think the other permissions are only requirements for this test

File: modules/apps/account/account-test/src/testFunctional/tests/AccountRoles.testcase [hunk 2]
```diff
@@ -188,4 +188,91 @@ definition {
...
+			Accounts.openAccountsAdmin();
+
+			LexiconEntry.gotoEntry(rowEntry = "Account Name");
+
+			Accounts.addAssigneeToRoleViaUsersTab(
+				accountRoleTitle = "Account Specific Role",
+				accountUserName = "userfn1 userln1");
+		}
+
+		task ("Then he can add account roles in Account Name") {
```
Comment (patriciajeanperez) [inline modules/apps/account/account-test/src/testFunctional/tests/AccountRoles.testcase:255]: @melodywu Please make sure task descriptions are up to date

**Review by patriciajeanperez — COMMENTED** [review]
@melodywu It might be better to have a separate macro for editing an account role. It might be hard for future writers to see that this add macro does an edit too just by reading the macro names. It also makes it easier to see what the test does just by reading the test case. What do you think?

**Comment by melodywu** [comment]
> @melodywu It might be better to have a separate macro for editing an account role. It might be hard for future writers to see that this add macro does an edit too just by reading the macro names. It also makes it easier to see what the test does just by reading the test case. What do you think?

Got it , Yesterday I'v thought about this issue, I remember that in our poshi we usually write add and edit together,so... But separate them is more clear indeed.

File: modules/apps/account/account-test/src/testFunctional/tests/AccountRoles.testcase (hunk 2 above)
Comment (melodywu) [reply modules/apps/account/account-test/src/testFunctional/tests/AccountRoles.testcase:255]: Sorry&Done

**Pull request closed without merging**